subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/configurations/DefaultConfiguration.java

```diff
@@ -480,11 +482,24 @@ public void run(BuildOperationContext context) {
                 dependencyResolutionListeners.getSource().afterResolve(incoming);
                 // Discard listeners
                 dependencyResolutionListeners.removeAll();
+                context.setResult(new ResolveDependenciesBuildOperationType.Result() {
+                    @Override
+                    public ResolvedComponentResult getRootComponent() {
+                        return incoming.getResolutionResult().getRoot();
+                    }
+                });
             }
 
             @Override
             public BuildOperationDescriptor.Builder description() {
-                return BuildOperationDescriptor.displayName("Resolve dependencies of " + identityPath).progressDisplayName("Resolve dependencies " + identityPath);
+                return BuildOperationDescriptor.displayName("Resolve dependencies of " + identityPath)
+                    .progressDisplayName("Resolve dependencies " + identityPath)
```

Comment:
This is not new, but bothered me before: Missing "of"

Reply:
fixed this.